aiida_pseudo/groups/family/pseudo_dojo.py

```diff
@@ -269,6 +271,10 @@ def parse_djrepos_from_folder(cls, dirpath, pseudo_type):
             if not os.path.isfile(filepath):
                 raise ValueError(f'dirpath `{dirpath}` contains at least one entry that is not a file')
 
+            if filename.split('.')[-1] not in cls._pseudo_repo_file_extensions:
```

Comment:
If you could just add a comment here why this is necessary, this should be good to go